packages/ckeditor5-image/src/imageutils.ts

```diff
@@ -211,6 +214,13 @@ export default class ImageUtils extends Plugin {
 		return this.isImage( selectedElement ) ? selectedElement : selection.getFirstPosition()!.findAncestor( 'imageBlock' );
 	}
 
+	/**
+	 * Returns an image widget editing view based on the passed image view.
+	 */
+	public getImageWidgetFromImageView( imageView: ViewElement ): ViewContainerElement | null {
+		return imageView.findAncestor( { classes: IMAGE_WIDGETS_CLASSES_MATCH_REGEXP } ) as ViewContainerElement;
```

Comment:
```suggestion
		return imageView.findAncestor( { classes: IMAGE_WIDGETS_CLASSES_MATCH_REGEXP } ) as ( ViewContainerElement | null );
```
It doesn't make a difference, though. But the cast is to the wrong type. Another alternative is to cast to `any` in order  to not repeat yourself.